lib/presentation/routes/router.dart

```diff
@@ -1,6 +1,11 @@
 import 'package:auto_route/auto_route.dart';
 import 'package:flutter_template/presentation/routes/router.gr.dart';
 
+/// 💡 Tip: Both `~Screen`, `~Page` will be treated as `~Route` by [AutoRoute].
+///
+/// Consider using `~Screen` as screen naming, and `~Page` if it is part of screen view,
+/// for example, Bottom navigation child views, tab views, etc.
```

Comment:
Was just going to mention this. I think this has to be documented somewhere in Confluence.